CONTRIBUTING.md

```diff
@@ -168,6 +168,11 @@ Set up your Firebase project as follows:
    1. Go to the Firebase Console, and select **Realtime Database** from the **Build** menu.
    2. Click on the **Create Database** button. You can choose to set up the Realtime Database
    either in the locked mode or in the test mode.
+
+   > **Note:** Integration tests are not ran against the default Realtime Database reference and are
```

Comment:
This note works great but needs some nitting from a TW: use "run" instead of "ran", and check the spelling of reference in line 174.

With those nits, LGTM.

Reply:
Done!